repo/packages/Q/quobyte/0/config.json

```diff
@@ -0,0 +1,110 @@
+          "type": "string"
+        }
+      },
+      "required": ["master"],
+      "type": "object"
+    },
+    "quobyte": {
+      "type": "object",
+      "description": "Quobyte Framework specific configuration properties",
+      "properties": {
+        "cpus": {
+          "default": 1.0,
+          "description": "CPU shares to allocate to each Quobyte scheduler instance.",
+          "minimum": 0.1,
+          "type": "number"
+        },
+        "mem": {
+          "default": 1024.0,
+          "description": "Memory (MB) to allocate to each Quboyte scheduler instance.",
+          "minimum": 768.0,
+          "type": "number"
+        },
+        "instances": {
+          "default": 1,
+          "description": "Number of Quobyte framework instances to run.",
+          "minimum": 1,
+          "type": "integer"
+        },
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "quobyte"
+        },
+        "zk": {
+          "type": "string",
+          "description": "Name or IP address of the zookeeper server.",
+          "default": "master.mesos:2181"
+          },
+        "port": {
+          "type": "string",
+          "description": "Port to which the framework's HTTP server is bound.",
+          "default": "7878"
+          },
+        "docker_image": {
+          "type": "string",
+          "description": "Docker image to be used for the Quobyte services.",
+          "default": "quobyte-server"
+          },
+        "framework_image": {
+          "type": "string",
+          "description": "Quobyte Mesos framework image version",
+          "default": "quobyte/quobyte-mesos:latest"
+          },
+        "registry_resources": {
+          "type": "string",
+          "description": "Resources for data service",
+          "default": "cpus:0.5;mem:2084;disk:8"
+          },
+        "metadata_resources": {
+          "type": "string",
+          "description": "Resources for data service",
+          "default": "cpus:1.0;mem:4096;disk:8"
+          },
+        "data_resources": {
+          "type": "string",
+          "description": "Resources for data service",
+          "default": "cpus:2.0;mem:3072;disk:15"
+          },
+        "api_resources": {
+          "type": "string",
+          "description": "Resources for data service",
+          "default": "cpus:1.0;mem:4096;disk:3"
+          },
+        "webconsole_resources": {
+          "type": "string",
+          "description": "Resources for data service",
+          "default": "cpus:2.0;mem:3072;disk:3"
+          },
+        "framework_url": {
```

Comment:
I was expecting the framework config to consume this in order to publish the URL in FrameworkInfo. I don't see that happening. What's this used for then?